Notes.Backend/Notes.Application/Interfaces/INotesDbContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Notes.Domain;
+using Microsoft.EntityFrameworkCore;
+
+namespace Notes.Application.Interfaces
+{
+    public interface INotesDbContext
```

Comment:
Instead of using interface for context I would use repository pattern that incapsulates dbcontext